translations/al.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return array_merge(require_once __DIR__ . '/global.php', [
+    'Bitcoin 2.0:' => 'Bitcoin 2.0:',
```

Comment:
This is just a starting point to show how translations should be done. For now, these will render in English.